portfolio/src/main/webapp/index.html

```diff
@@ -15,9 +15,12 @@ <h3 id="text">This is my portfolio! I will present some of my most important asp
       <div id="selfie-container">
         <img src="Images/security.jpg" id="selfie" alt="Photo of Andrés in a small mexican town."/>
       </div>
-      <p id="text">Click here to get a random language that I speak:</p>
-      <button onclick="addRandomLanguage()">Click!</button>
-      <div id="language-container"></div>
+      <div>
+        <p id="text">Click here to get a random language that I speak:</p>
+        <button onclick="addRandomLanguage()">Click!</button>
+        <div id="language-container"></div>
+      </div>
+      <div id="content-container"></div>
```

Comment:
It might be worth adding a comment above this div so it's clear what this is for!

Reply:
Done!